home/.vimrc

```diff
@@ -199,4 +195,9 @@ set foldlevelstart=99
 " https://github.com/elzr/vim-json#common-problems
 let g:vim_json_syntax_conceal = 0
 
+" putting this here so users can override anything specified above
+if filereadable(glob("~/.vimrc.local"))
+  source ~/.vimrc.local
+endif
+
 runtime! macros/matchit.vim
```

Comment:
I assume there's a reason to have `matchit.vim` last, but I'm mentioning it just in case that's an oversight.

Reply:
@benjaminoakes it was an oversight. I've fixed it.